georiviere/river/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework_gis.fields import GeometryField
+from rest_framework_gis.serializers import GeoFeatureModelSerializer
+from rest_framework.serializers import ModelSerializer
+
+from georiviere.river.models import Stream
+
+
+class StreamGeojsonSerializer(GeoFeatureModelSerializer):
+    # Annotated geom field with API_SRID
+    api_geom = GeometryField(read_only=True, precision=7)
```

Comment:
```suggestion
    api_geom = GeometryField(read_only=True, precision=7)
```
use mapentity geojson setting precision for this